### DISCUSSION TITLE
Updated Windows 11 and now Fan Control won't run

### DISCUSSION THREAD

**Discussion by Eeshen** [Rem0o/FanControl.Releases#766] [General]
Fan Control was working just fine, Windows 11 did an update, now every time I click on Fan Control, it asks for permission, I say, but nothing happens, FC does not load.

Any suggestions?

Thanks

**Comment by xhul-dev** [comment]
Some have reported that uninstalling the update is possible, and fixes the issue.
Another thing worth a shot, create a shortcut that points to FanControl.exe, then check "run as admin" in the shortcut properties.

**Comment by Eeshen** [comment]
I've tried the Run as Admin, didnt work.   This Win update causes all kinds of headaches for me !

**Reply by xhul-dev** [reply]
> I've tried the Run as Admin, didnt work. This Win update causes all kinds of headaches for me !

I'll write a tool after work, that may solve the problem (for Fan Control), would you be ok to test it?

**Comment by Eeshen** [comment]
I just reversed one of the Windows updates and Fan Control is working again.....probably until Windows updates again.

**Comment by Blackwolf2022** [comment]
You can mitigate this issue by re-enabling .NET Framework 3.5 and the Windows Communication Foundation in Windows Features. For instructions, please see [Enable the .NET Framework 3.5 in Control Panel](https://docs.microsoft.com/dotnet/framework/install/dotnet-35-windows#enable-the-net-framework-35-in-control-panel).

**Comment by Eeshen** [comment]
I believe it was:
2022-05 Cumulative Update for Windows 11 for x64-based Systems (KB5013943)

.....and looks like Windows will be installing it soon.

**Reply by Blackwolf2022** [reply]
No, you're wrong! the problem is the netframework, just enable them again and it works!

**Comment by Eeshen** [comment]
Thank you so much for this !!
